.github/workflows/_linux_ut.yml

```diff
@@ -142,3 +144,31 @@ jobs:
             test_cmd="${test_cmd} test_xpu.py"
           fi
           eval $test_cmd
+
+      - name: Upload Inductor XPU UT Data
+        uses: actions/upload-artifact@v4
+        with:
+          name: Inductor-XPU-UT-Data-${{ github.event.pull_request.number || github.sha }}
```

Comment:
Change the name to `Torch-XPU-UT-Log-xxx`. And we can move the upload step after the result check step with `if: always()`